org.milkytracker.MilkyTracker.yaml

```diff
@@ -0,0 +1,65 @@
+app-id: org.milkytracker.MilkyTracker
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+command: milkytracker
+finish-args:
+  - --device=dri
+  - --share=ipc
+  - --socket=pulseaudio
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --filesystem=xdg-music
+
```

Comment:
Cleanups:

```
- "*.a"
- /include
- /lib/pkgconfig
- /share/aclocal
- /share/doc
- /share/man
- /share/rtmidi

Reply:
Thanks! I thought that some of them are added by default. I am not sure about the `/share/rtmidi` one. Is it ok to remove this dir? Isn't it used by rtmidi?

Reply:
Seems to be only CMake source files.

Reply:
Added. Thanks!